client/src/extension.ts

```diff
@@ -25,6 +25,7 @@ export interface IClientSettings {
   elmPath: string;
   elmTestPath: string;
   trace: { server: string };
+  elmAnalyseTrigger: string;
```

Comment:
Could go with the string union type here?

Reply:
I'm not sure if that would work, but I can try.

Reply:
Seems to work